ddcz/views/postal_service.py

```diff
@@ -67,7 +68,7 @@ def handle_postal_service_post_request(request):
     if fid == FORM_SEND:
         try:
             Letter.objects.create(
-                receiver=UserProfile.objects.get(id=request.POST.get("whom")).nick,
+                receiver=request.POST.get("whom"),
```

Comment:
Why? This is both XSS security hole as well as an option to send a letter to a nonexisting user